### DISCUSSION TITLE
[2021.08.31] 없어진 기록 찾기

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#86] [SQL]
[없어진 기록 찾기](https://programmers.co.kr/learn/courses/30/lessons/59042)

**Comment by pyunji** [comment]
[없어진 기록 찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59042.sql)

**Comment by hyk339** [comment]
[없어진 기록 찾기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/pr_59042.sql)

**Comment by leechang4** [comment]
[없어진 기록 찾기 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059042.sql)

**Reply by leechang4** [reply]
RIGHT JOIN = 오른쪽 테이블은 무조건 조회되며, 왼쪽 테이블에 조건이 맞는 것이 있다면 같이 보여준다.

**Comment by chaeyeon4u** [comment]
[없어진 기록 찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59042.sql)
